multiomics/README.md

```diff
@@ -0,0 +1,50 @@
+# Multiomics Datasets
+
+This repository contains a collection of publicly available multiomics datasets that have been slightly modified for use
+with the [PyKale](https://github.com/pykale/pykale) library.
+
+## 1. TCGA-BRCA
+
+The Cancer Genome Atlas Breast Invasive Carcinoma (TCGA-BRCA) is a multiomics dataset for subtype classification of 
+breast cancer (i.e., normal-like, basal-like, HER2-enriched, Luminal A, and Luminal B subtypes). The original version of
+this dataset can be obtained from the TCGA program through [Broad GDAC Firehose](https://gdac.broadinstitute.org/).
+
+Here, we provide the proprocessed version of TCGA-BRCA dataset in accordance with the **Preprocessing** section of the 
```

Comment:
"proprocessed" should be preprocessed, right? Install a spell check should fix this kind of problem.

Reply:
"TCGA-BRCA dataset" --> "the TCGA-BRCA dataset"

Reply:
Corrected these typos in the revised version.